pkgs/os-specific/bsd/openbsd/pkgs/include/no-perms.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/include/Makefile b/include/Makefile
+index a2d539f1c3b..c256c2e2b43 100644
+--- a/include/Makefile
++++ b/include/Makefile
+@@ -109,7 +109,6 @@ includes:
+ 		echo installing in ${.CURDIR}/$$i; \
+ 		cd ${.CURDIR}/$$i && ${RUN_MAKE}; \
+ 	done
+-	chown -RP ${BINOWN}:${BINGRP} ${DESTDIR}/usr/include
```

Comment:
Could we avoid the patch by setting BINOWN and BINGRP to the current user and group in a preBuild?

Reply:
fixed